test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -291,8 +291,9 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 	})
 
 	Context("RPS configuration", func() {
-		It("Should have the correct RPS configuration", func() {
-			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved != nil {
+		It("[test_id:54190] Should have the correct RPS configuration", func() {
+			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved == nil {
+				Skip("Test Skipped due nil Reserved cpus")
```

Comment:
the `return` is now redundant (code will never be reached) but we can clean up later.
We now require both fields to be != nil.
Question: is it OK to run the test if CPU != nil && reserved == nil ? If I'm not mistaken, the old code allowed that (intentionally or by side effect?)

Reply:
I had the same query, Actually i think the if condition is not required, because we can't have profile without reserved cpus in the first place.

Reply:
fair enough